### DISCUSSION TITLE
dotnet format example

### DISCUSSION THREAD

**Discussion by sander1095** [alirezanet/Husky.Net#41] [Q&A] [answered]
Hello! I would really appreciate an example in the docs about formatting code on pre commit with dotnet format. I think it'd be easier to get started with the library!

**Comment by alirezanet** [comment, marked as answer]
Hi,
Thanks for the feedback. good idea I'll try to add some examples to the docs soon but for now, you can check the husky repo to get some ideas. 

task-runner:
https://github.com/alirezanet/Husky.Net/blob/69bb3036fb76f1418bb900034dca4c784e2d6c4e/.husky/task-runner.json#L15-L21

pre-commit hook:
https://github.com/alirezanet/Husky.Net/blob/69bb3036fb76f1418bb900034dca4c784e2d6c4e/.husky/pre-commit#L1-L4
